gulpfile.js

```diff
@@ -143,9 +143,7 @@ var jsTasks = function(filename) {
 var writeToManifest = function(directory) {
   return lazypipe()
     .pipe(gulp.dest, path.dist + directory)
-    .pipe(function() {
-      return gulpif('**/*.{js,css}', browserSync.reload({stream:true}));
-    })
+    .pipe(browserSync.stream, {match: '**/*.{js,css}'})
```

Comment:
JavaScript files should be a full page reload. CSS injecting is working properly over here after omitting the inline reload.